lib/docs/src/components/table.stories.tsx

```diff
@@ -280,6 +292,16 @@ export const SelectableSingle = (): JSX.Element => {
 	);
 };
 
+Utils.story(SelectableSingle, {
+	desc: `
+Moai also provides user with single choice option. By adding the keyword
+\`radioGroupName\` to the object, checkboxes will be replaced by [radio buttons][1],
+assure that user can only choose one row from table's rows.
```

Comment:
Like above, "radioGroupName" is not the condition. The condition is the type of selectable.selected state. If it's a string, then radioGroupName must be defined.